static/app/views/issueList/searchBar.tsx

```diff
@@ -121,15 +123,27 @@ function IssueListSearchBar({organization, tags, onClose, ...props}: Props) {
           : undefined,
         statsPeriod: pageFilters.datetime.period,
       };
+      const eventsDatasetValues = await fetchTagValues({
```

Comment:
Instead of making sequential requests, we can make them in parallel with `Promise.all`. Since the requests aren't dependent on each other, it makes sense to do it in parallel.